x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/timeline/timeline.tsx

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import React, { FunctionComponent, useMemo } from 'react';
+import {
+  EuiText,
+  EuiIcon,
+  EuiIconProps,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiIconTip,
+} from '@elastic/eui';
+
+import { PhasesExceptDelete } from '../../../../../../common/types';
+import { useFormData } from '../../../../../shared_imports';
+
+import { FormInternal } from '../../types';
+
+import {
+  calculateRelativeTimingMs,
+  normalizeTimingsToHumanReadable,
+  PhaseAgeInMilliseconds,
+} from '../../lib';
+
+import './_timeline.scss';
```

Comment:
Since you're importing this file directly into JS, the pattern is to not use the leading `_` in the file name.
```suggestion
import './timeline.scss';
```